src/vega.schema.ts

```diff
@@ -141,25 +141,24 @@ export type VgScale = Pick<BaseScale, 'type'> & {
   zero?: boolean | SignalRef; // LogScale only allow false, making the intersection type overly strict
 } & Omit<
     // Continuous
-
-    LinearScale &
-      LogScale &
-      SymLogScale &
-      Partial<PowScale> & // use partial so exponent is not required
-      SqrtScale &
-      IdentityScale &
-      TimeScale &
-      // discretizing
-      QuantileScale &
-      QuantizeScale &
-      ThresholdScale &
-      BinOrdinalScale &
-      // sequential
-      SequentialScale &
-      // discrete
-      BandScale &
-      PointScale &
-      OrdinalScale,
+    Omit<LinearScale, 'type'> &
+      Omit<LogScale, 'type'> &
+      Omit<SymLogScale, 'type'> &
+      Omit<Partial<PowScale>, 'type'> & // use partial so exponent is not required
+      Omit<SqrtScale, 'type'> &
+      Omit<IdentityScale, 'type'> &
+      Omit<TimeScale, 'type'> &
+      // Discretizing
+      Omit<QuantileScale, 'type'> &
+      Omit<QuantizeScale, 'type'> &
+      Omit<ThresholdScale, 'type'> &
+      Omit<BinOrdinalScale, 'type'> &
+      // Sequential
+      Omit<SequentialScale, 'type'> &
+      // Discrete
+      Omit<BandScale, 'type'> &
+      Omit<PointScale, 'type'> &
+      Omit<OrdinalScale, 'type'>,
```

Comment:
Why `omit` type individually given that we already omit in the outer `Omit`?

Reply:
Good catch. The previous code was wrong because the type can never be satisfied if you don't omit the `type`. Typescript fixed a bug that previously existed.